Create pm-unanet-tasks.md

farooqzakhilwal opened the pull request:
[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/farooqzakhilwal-patch-1/ ![Documentation Status](https://readthedocs.org/projects/civicactions-handbook/badge/?version=farooqzakhilwal-patch-1)

[//]: # (rtdbot-end)

farooqzakhilwal committed:
Create pm-unanet-tasks

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra reviewed (changes requested):
@farooqzakhilwal the new file will need extension `.md` can you add that?

farooqzakhilwal committed:
Rename pm-unanet-tasks to pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra committed:
Merge branch 'master' into farooqzakhilwal-patch-1

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Merge branch 'master' into farooqzakhilwal-patch-1

farooqzakhilwal committed:
Update pm-unanet-tasks.md

hhartingrex committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

hhartingrex committed:
Update pm-unanet-tasks.md

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Merge branch 'master' into farooqzakhilwal-patch-1

dmundra reviewed (changes requested):
@farooqzakhilwal could you upload the images to the same branch in the `/docs/images folder` and then reference them in the article. See this example https://github.com/CivicActions/handbook/pull/864/files#diff-99b0b0ed3d6aa5c6ada888fc3e6fae6c1accb99d09554af89f767fd22050d0f8R64?

farooqzakhilwal committed:
Add files via upload

farooqzakhilwal committed:
Add files via upload

farooqzakhilwal committed:
Delete Plan with Grid 1.png

farooqzakhilwal committed:
Delete Plan with Grid 2.png

farooqzakhilwal committed:
Delete Plan with Grid 3.png

farooqzakhilwal committed:
Delete Plan with Grid 4.png

farooqzakhilwal committed:
Delete Plan with Grid 5.png

farooqzakhilwal committed:
Delete Plan with Grid 6.png

farooqzakhilwal committed:
Delete Planned Utilization 1.png

farooqzakhilwal committed:
Delete Planned Utilization 2.png

farooqzakhilwal committed:
Delete Plan with Grid 7.png

farooqzakhilwal committed:
Delete Planned Utilization 3.png

farooqzakhilwal committed:
Delete Plans vs Actuals 1.png

farooqzakhilwal committed:
Delete Plans vs Actuals Report 2.png

farooqzakhilwal committed:
Delete Approving Expenses 1.png

farooqzakhilwal committed:
Delete Plan-with-Grid-1.png

farooqzakhilwal committed:
Add files via upload

farooqzakhilwal committed:
Update pm-unanet-tasks.md

farooqzakhilwal committed:
Add files via upload

farooqzakhilwal committed:
Update pm-unanet-tasks.md

dmundra reviewed (commented):
Thanks for making the changes @farooqzakhilwal. Do you need my approval on the content to move this forward?

hhartingrex committed:
Update pm-unanet-tasks.md

hhartingrex committed:
Update pm-unanet-tasks.md

1/ I think we should remove the 2997, last four digits of the CA CC
2/ I think we should remove Rachael's name
3/ The a, b, c lists are sometimes hard to read
Otherwise, VERY nice. I look forward to building on this page. Thank you!

hhartingrex committed:
Update pm-unanet-tasks.md

Adjusted the sub-level bullet lists

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Update pm-unanet-tasks.md

farooqzakhilwal committed:
Merge branch 'master' into farooqzakhilwal-patch-1

farooqzakhilwal committed:
Merge branch 'master' into farooqzakhilwal-patch-1

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,255 @@
+
+      i. People/List/Type last name in Last Name field and click blue > then List
+      ii. Click on the dollar icon
+      iii. Click on 'Completed Expense Requests' to view
```
@farooqzakhilwal the formatting of the below bullet lists (i. ii. iii.) is showing up like a code block here https://handbook.civicactions.com/en/farooqzakhilwal-patch-1/070-project-management/pm-unanet-tasks/. Maybe reduce the indentation or use a. b. c.? Might have to try different combinations.

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,255 @@
+          2. Check project/expense type
+          3. Check the receipt date (should match date entered in Unanet)
+          4. Check the receipt amount (should match amount entered in Unanet)
+          5. Check the credit card number does not end in ****. If it does, then the receipt needs to be associated with our company credit card and not reimbursed. If it is the company credit card, click on 'disapprove' and ask the  requestor to resubmit with '**** Visa' for Payment Method (on the Details page)
```
@farooqzakhilwal same indentation problem here I believe.

farooqzakhilwal replied:
@dmundra should be good now?

dmundra replied:
Na it is still showing up in a code snippet format. Maybe try the removing an additional 4 spaces of indentation?

dmundra reviewed (changes requested):
Some minor changes.

farooqzakhilwal committed:
Update pm-unanet-tasks.md

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Update pm-unanet-tasks.md

farooqzakhilwal committed:
Merge branch 'master' into farooqzakhilwal-patch-1

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,241 @@
+   1. Check the amount, project type and expense type are correct
+   1. Click on attachments to open the receipt and double check the details
+   1. Checklist
+      1. Receipt should be attached
```
@farooqzakhilwal looks like unfortunately lists 3-levels or deeper don't render well, see https://handbook.civicactions.com/en/farooqzakhilwal-patch-1/070-project-management/pm-unanet-tasks/

They all appear as:

> Checklist 1. Receipt should be attached 1. Check project/expense type 1. Check the receipt date (should match date entered in Unanet) 1. Check the receipt amount (should match amount entered in Unanet) 1. Check the credit card number does not end in xxxx. If it does, then the receipt needs to be associated with our company credit card and not reimbursed. If it is the company credit card, click on "disapprove" and ask the requestor to resubmit with "x Visa" for Payment Method (on the Details page) 1. If there are any problems with the report, click on "disapprove" and let the requestor know why you have disapproved it 1. If all looks good, click on "Approve"

An alternate solution is using sub-headings and then having lists under those.

dmundra replied:
Sorry the handbook rendering can be a pain.

farooqzakhilwal replied:
I created a list under the checklist instead, hopefully, this resolves it.

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,241 @@
+         1. If there are any problems with the report, click on "disapprove" and let the requestor know why you have disapproved it
+         1. If all looks good, click on "Approve"
+
+**Expense report and request statuses and what they mean:**
```
I just noticed this, maybe it should be a heading?

dmundra reviewed (changes requested):
Sorry @farooqzakhilwal more changes for you.

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,241 @@
+   1. To check this by viewing the person expense approval history:
+      1. People/List/Type last name in Last Name field and click blue > then List
+      1. Click on the dollar icon
+      1. Click on "Completed Expense Requests" to view
```
Both these 3 level checklists are not showing up 😞

farooqzakhilwal replied:
@dmundra I think the sub-headings should have helped solve this issue

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,241 @@
+      - Check the receipt amount (should match amount entered in Unanet)
+      - Check the credit card number does not end in xxxx. If it does, then the receipt needs to be associated with our company credit card and not reimbursed. If it is the company credit card, click on "disapprove" and ask the requestor to resubmit with "x Visa" for Payment Method (on the Details page)
+      - If there are any problems with the report, click on "disapprove" and let the requestor know why you have disapproved it
+      - If all looks good, click on "Approve"
```
Unfortunately @farooqzakhilwal that didn't work either https://handbook.civicactions.com/en/farooqzakhilwal-patch-1/070-project-management/pm-unanet-tasks/

farooqzakhilwal replied:
@dmundra i've updated this part

dmundra reviewed (changes requested):
Let me know if you want me to take a crack at the changes. I was thinking we can third level headers like:

```
### 1. Find your approvals

checklist...

### 2. Review the REQUEST

checklist...

### 3. Review the REPORT

checklist...
```

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

farooqzakhilwal committed:
Merge branch 'master' into farooqzakhilwal-patch-1

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,246 @@
+1.  To check this quickly:
+    1. Click on the magnifying glass next to the expense report
+    1. Scroll down to Approval History
+    1. A report that has previously been a request will show "PreApproving" and "PreApproved" statuses in the history. If it has never been a request, you will only see "Approving" and/or "Completed". (See below for explanation of statuses and an example of an expense request and report approval history).
```
@farooqzakhilwal one last change, reduce the indentation here my one space and that should fix it. If you look at https://handbook.civicactions.com/en/farooqzakhilwal-patch-1/070-project-management/pm-unanet-tasks/ you can see that the sub-list is all one line instead of a list.

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,246 @@
+1.  To check this by viewing the person expense approval history:
+    1. People/List/Type last name in Last Name field and click blue > then List
+    1. Click on the dollar icon
+    1. Click on "Completed Expense Requests" to view
```
Remove one space from these as well.

dmundra commented on docs/070-project-management/pm-unanet-tasks.md:
```diff
@@ -0,0 +1,246 @@
+    1. Check the receipt amount (should match amount entered in Unanet)
+    1. Check the credit card number does not end in xxxx. If it does, then the receipt needs to be associated with our company credit card and not reimbursed. If it is the company credit card, click on "disapprove" and ask the requestor to resubmit with "x Visa" for Payment Method (on the Details page)
+    1. If there are any problems with the report, click on "disapprove" and let the requestor know why you have disapproved it
+    1. If all looks good, click on "Approve"
```
Remove one space from these as well.

dmundra reviewed (changes requested):
One more set of changes and think you are good.

farooqzakhilwal committed:
Update pm-unanet-tasks.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra committed:
Update pm-unanet-tasks.md

dmundra reviewed (approved):
Okay I think it is good in terms of formatting.

dmundra committed:
Merge branch 'master' into farooqzakhilwal-patch-1
